README.md

```diff
@@ -15,27 +15,26 @@ release](https://img.shields.io/badge/docs-release-blue.svg)](https://rpolars.gi
 dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://rpolars.github.io/dev)
 <!-- badges: end -->
 
-The goal of this project is to bring the blazingly fast
-[Polars](https://www.pola.rs/) data manipulation library to R. The
-underlying computation engine is written in Rust and this R
-implementation has no other dependencies than R itself (≥ 4.1.0).
+The **polars** package for R gives users access to [a lightning
+fast](https://duckdblabs.github.io/db-benchmark/) Data Frame library
+written in Rust. [Polars](https://www.pola.rs/)’ embarrassingly parallel
+execution, cache efficient algorithms and expressive API makes it
+perfect for efficient data wrangling, data pipelines, snappy APIs and so
+much more. Polars also supports “streaming mode” for out-of-memory
+operations. This allows users to analyze datasets many times larger than
+RAM.
 
 Documentation can be found on the **r-polars**
 [homepage](https://rpolars.github.io).
 
 The primary developer of the upstream Polars project is Ritchie Vink
-([@ritchie46](https://github.com/ritchie46)). This R port is maintained
-by Søren Welling ([@sorhawell](https://github.com/sorhawell)), together
-with other
+(\[@ritchie46\](<https://github.com/ritchie46>)). This R port is
+maintained by Søren Welling
+(\[@sorhawell\](<https://github.com/sorhawell>)) and
```

Comment:
These lines seems broken?

Reply:
My `knitr` behaves very weirdly and escapes these links automatically. However, I have not modified the original links in `.Rmd` and they look fine. So I just edited the `.md` files in d399206

Reply:
Do you use `devtools::build_readme()` function?

Reply:
I get the same escaping with that function too.

Reply:
Okay, that's odd.......

Note that escapes have been added in several other places.
If you don't mind, I would be happy to help render after you have finished editing the Rmd file.

Reply:
Yes, very odd. Thanks for rendering! I'm done with editing now.

Reply:
I render the file.

Do you have the latest versions of rmarkdown and knitr installed?

Reply:
yes i do. 2.21 and 1.42. But don't worry about it. I'll try to fix this on my machine when I find time later.